ui/packages/catalog-ui-search/src/main/webapp/component/editor/editor.view.jsx

```diff
@@ -56,7 +56,39 @@ module.exports = Marionette.LayoutView.extend({
   setDefaultModel() {
     //override
   },
-  template,
+  template() {
+    const canEdit = this.model.reduce((canEdit, result) => {
```

Comment:
What is this reducing over? A list of metacards?

Reply:
Yeah.

Reply:
same comment on the `reduce` vs. `every`.